Week07/lesson14/classwork/responsive_articles_with_menu/css/styles.css

```diff
@@ -0,0 +1,216 @@
+/* Box Model Hack */
+* {
+ box-sizing: border-box;
+}
+
+/* Clearfix hack */
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-size: 13px;
+  background-color: #F6F0E4;
+}
+
+/* Basic/Mobile Article Styling: */
+
+nav {
+  background-color: #7BAFA4;
+}
+
+nav article {
+  padding: 0 20px;
+}
+
+menu {
+  float: right;
+}
+
+.menubutton {
+  position: relative;
+  background: white;
+  height: 5px;
+  width: 30px;
+  border-radius: 9px;
+  margin-bottom: 2px;
+}
+
+ul {
+  padding: 0;
+  margin: 0 20px;
+  transition: max-height 0.5s;
+  max-height: 0px;
+  overflow: hidden;
+}
+
+.visible {
+  max-height: 606px;
+}
+
+li {
+  padding: 10px;
+  border-bottom: 1px solid #F6F0E4;
+}
+
+li:first-child {
+  border-top: 1px solid #F6F0E4;
+}
+
+li:last-child {
+  margin-bottom: 20px;
+}
+
+li:hover, li:focus, menu:hover > div, menu:focus > div {
+  background-color: #488A7C;
+}
+
+a, a:visited {
+  text-decoration: none;
+  color: white;
+}
+
+main {
+  margin: 0 auto;
+}
+
+article {
+  padding: 20px;
+}
+
+h2 {
+  font-family: 'PT Serif', serif;
+  font-size: 2em;
+  margin: 20px 0;
+}
+
+p {
+  margin: 0;
+}
+
+img {
+  display: block;
+  width: 100%;
+}
+
+/* Responsive Article Styling: */
+
+@media (min-width: 480px) { /* mobile-landscape */
+  /* limit <main> to 440px and do necessary styling to match screenshot
+  Use classes and elements present in HTML as clues */
+  main {
+    width: 440px;
+    font-size: 0px;
+  }
+
+  h2 {
+    font-size: 2em;
+    margin-top: 0;
+  }
+
+  .right-aligned {
+    float: right;
+  }
+
+  .left-aligned {
+    margin-right: 6%;
+  }
+
+  .right-aligned, .left-aligned {
+    font-size: 13px;
+    display: inline-block;
+    width: 47%;
```

Comment:
This is one of those cases where it's better to do the maths than end up with weird percentages. If `<main>` is `440px` wide, and has `20px` of padding on either side and you want a `20px` gutter between two columns:

`( container - padding_on_either_side - gutter(s) ) / 2`
`( 440 - 40 - 20 ) / 2`

... which is `190px`. Which leads to much nicer CSS:

```
.left-aligned {
  margin-right: 20px;
  float: left;
}

.right-aligned {
  width: 190px;
  float: right;
}

.right-aligned, .left-aligned {
  width: 190px;
}
```

... also by floating both elements (as mentioned in my comment above), you don't need to worry about that pesky newline character